src/MICmdBase.cpp

```diff
@@ -190,7 +190,8 @@ void CMICmdBase::SetError(const CMIUtilString &rErrMsg) {
   m_cmdData.strErrorDescription = rErrMsg;
   m_cmdData.bCmdExecutedSuccessfully = false;
 
-  const CMICmnMIValueResult valueResult("msg", CMICmnMIValueConst(rErrMsg));
+  const CMICmnMIValueResult valueResult(
```

Comment:
I'd make CMICmnMIValueResult's constructor take a string (and, maybe, a boolean `do_escape`) parameter to avoid such errors in the future.